assets/js/components/ReportError.js

```diff
@@ -151,22 +160,37 @@ export default function ReportError( { moduleSlug, error } ) {
 		} );
 	}, [ dispatch, retryableErrors ] );
 
+	const showRequestAccessURL =
+		requestAccessURL && hasInsufficientPermissionsError;
+
 	return (
 		<CTA title={ title } description={ description } error>
+			{ showRequestAccessURL && (
+				<Button href={ requestAccessURL } target="_blank">
+					{ __( 'Request access', 'google-site-kit' ) }
+				</Button>
+			) }
 			{ showRetry ? (
 				<Fragment>
 					<Button onClick={ handleRetry }>
 						{ __( 'Retry', 'google-site-kit' ) }
 					</Button>
 					<span className="googlesitekit-error-retry-text">
-						{ __( 'Retry didn’t work?', 'google-site-kit' ) }{ ' ' }
+						{ __( "Retry didn't work?", 'google-site-kit' ) }{ ' ' }
```

Comment:
Is this an unnecessary change - perhaps made by the editor?

Reply:
My VS Code was complaining about it. I have reverted now.